packages/components/src/VirtualizedList/CellTitle/CellTitleActions.scss

```diff
@@ -40,7 +40,16 @@ $tc-list-title-dropup-menu-triangle: '\25bc';
 
 	:global {
 		.btn-link {
-			padding: 0 $padding-small;
+			padding: 0 $padding-smaller;
```

Comment:
I approve but guideline needs to be updated. Can you see that with the UX please ?

Reply:
It's relative to the TUX no ?